app/interfaces/import.ts

```diff
@@ -47,14 +49,40 @@ export interface IImportFile {
   content: string[];
   withHeader?: boolean;
   delimiter?: string;
+  s3Config?: IS3Config;
+  sftpConfig?: ISftpConfig;
+  /** remote path */
+  path?: string;
+}
+
+export interface IS3Config {
+  region: string;
+  endpoint: string;
+  accessKey: string;
+  accessSecret: string;
+  bucket?: string;
+  token?: string;
+  key: string;
```

Comment:
what are the token and key?

Reply:
they are the parameters of the importer, key is the file path